content/en/_redirects

```diff
@@ -337,4 +337,19 @@ docs/started/requirements/                    /docs/started/getting-started/
 /docs/components/pipelines/v2/reference/api/kubeflow-pipeline-api-spec/     /docs/components/pipelines/reference/api/kubeflow-pipeline-api-spec/
 /docs/components/pipelines/v2/reference/sdk/                                /docs/components/pipelines/reference/sdk/
 /docs/components/pipelines/v2/run-a-pipeline/                               /docs/components/pipelines/user-guides/core-functions/run-a-pipeline/
-/docs/components/pipelines/v2/version-compatibility/                        /docs/components/pipelines/reference/version-compatibility/
\ No newline at end of file
+/docs/components/pipelines/v2/version-compatibility/                        /docs/components/pipelines/reference/version-compatibility/
+
+# Kubeflow Training V2 (https://github.com/kubeflow/training-operator/issues/2214)
+/docs/components/training/installation                       /docs/components/training/legacy-v1/installation
+/docs/components/training/explanation/fine-tuning            /docs/components/training/legacy-v1/explanation/fine-tuning
+/docs/components/training/reference/architecture             /docs/components/training/legacy-v1/reference/architecture
+/docs/components/training/reference/distributed-training     /docs/components/training/legacy-v1/reference/distributed-training
+/docs/components/training/reference/fine-tuning              /docs/components/training/legacy-v1/reference/fine-tuning
+/docs/components/training/user-guides/fine-tuning            /docs/components/training/legacy-v1/user-guides/fine-tuning
+/docs/components/training/user-guides/jax                    /docs/components/training/legacy-v1/user-guides/jax
+/docs/components/training/user-guides/job-scheduling         /docs/components/training/legacy-v1/user-guides/job-scheduling
+/docs/components/training/user-guides/mpi                    /docs/components/training/legacy-v1/user-guides/mpi
+/docs/components/training/user-guides/paddle                 /docs/components/training/legacy-v1/user-guides/paddle
+/docs/components/training/user-guides/prometheus             /docs/components/training/legacy-v1/user-guides/prometheus
+/docs/components/training/user-guides/tensorflow             /docs/components/training/legacy-v1/user-guides/tensorflow
+/docs/components/training/user-guides/xgboost                /docs/components/training/legacy-v1/user-guides/xgboost
```

Comment:
We discussed with @thesuperzapper how to structure the new docs, and he suggested that we put new user guides under
```/docs/components/training/user-guides-v2```
For example: 

That will allow us to redirect the V1 docs (e.g. TFJob) to the correct location.

What do you think about it @kubeflow/wg-training-leads ?

Reply:
@andreyvelich make sure you also redirect the "index" pages e.g.

- `/docs/components/training/explanation/` -> `/docs/components/training/legacy-v1/explanation/`
- `/docs/components/training/user-guides/` -> `/docs/components/training/legacy-v1/user-guides/`
- `/docs/components/training/reference/` -> `/docs/components/training/legacy-v1/reference/`

Also, I am not sure if it matters, but all other redirects use a trailing `/` slash.

Reply:
Good catch, let me fix that.